test/coverlet.core.tests/Samples/Samples.cs

```diff
@@ -122,6 +122,7 @@ public string HasSimpleUsingStatement()
             return value;
         }
 
+        [ExcludeFromCodeCoverage]
```

Comment:
I've got a doubt `GetBranchPoints_GeneratedBranches_DueToCachedAnonymousMethodDelegate_Ignored` is again the reason for `Empty`?

Reply:
If this method is instrumented, it no longer detects the pattern that would cause the branch to be ignored. Previously, the lack of instrumentation was caused by the method being located in the test assembly. The current behavior would only be a problem if a user attempted to instrument an assembly that is already instrumented, so it seemed safe to just make this small change that prevents the instrumentation process from making changes to this method.

Reply:
Clear thank's

Reply:
Could be useful thank's a lot